hedera-node/hedera-app/src/main/java/com/hedera/node/app/Hedera.java

```diff
@@ -897,6 +893,7 @@ public void onHandleConsensusRound(
             @NonNull final Round round, @NonNull final PlatformState platformState, @NonNull final HederaState state) {
         daggerApp.workingStateAccessor().setHederaState(state);
         daggerApp.handleWorkflow().handleRound(state, platformState, round);
+        daggerApp.platformStateAccessor().setPlatformState(platformState);
```

Comment:
Only the listener _currently_ needs access to `PlatformState` through this, right?

So it won't matter now if we set `platformState` before or after `handleRound()`.

But for clarity and just in case some handle component needs to use this accessor, setting the state first seems better.

Reply:
Yes, makes sense. Fixed. Thanks @tinker-michaelj